vg/vgimg/vgimg_test.go

```diff
@@ -0,0 +1,43 @@
+package vgimg_test
+
+import (
+	"bytes"
+	"log"
+	"testing"
+
+	"github.com/gonum/plot"
+	"github.com/gonum/plot/plotter"
+	"github.com/gonum/plot/vg"
+	"github.com/gonum/plot/vg/draw"
+	"github.com/gonum/plot/vg/vgimg"
+)
+
+func TestIssue179(t *testing.T) {
+	scatter, err := plotter.NewScatter(plotter.XYs{{1, 1}, {0, 1}, {0, 0}})
+	if err != nil {
+		log.Fatal(err)
+	}
+	p, err := plot.New()
+	if err != nil {
+		log.Fatal(err)
+	}
+	p.Add(scatter)
+	p.HideAxes()
+
+	c := vgimg.JpegCanvas{Canvas: vgimg.New(vg.Inch, vg.Inch)}
```

Comment:
Everytime I see a non-SI, non-ISO unit in scientific computing, I die a little inside.